main.py

```diff
@@ -45,7 +44,7 @@ def execute(self):
 
             self.execute_as_loop(30)  # 30-second interval.
         """
-        self.schedule.run_pending()
+        pass
```

Comment:
As execute() is empty, I think it would be nice to add a little note here explaining which method really executes this napp.